src/styles/style.css

```diff
@@ -0,0 +1,115 @@
+@import '../styles/stars.css';
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+  --white: #FFF;
+  --dark-blue: #060B35;
+  --gray: #616070;
+  --blue: #00ACDC;
+}
+
+.card {
+  width: 198px;
+  border-radius: 5px;
+  border: 1px solid var(--white);
+  background: var(--white);
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 36px;
+}
+
+.card__title {
+  color: var(--dark-blue);
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 18px;
+  margin: 0 16px 4px;
+}
+
+.title {
+  color: var(--gray);
+  font-size: 10px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 14px;
+  margin: 0 16px;
+}
+
+.review {
+  display: flex;
+  width: 166px;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin: 16px 16px 24px;
+}
+
+.reviews {
+  color: var(--dark-blue);
+  text-align: right;
+  font-size: 10px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 14px;
+  margin: 0;
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin: 0;
+  margin-right: 16px;
+  margin-left: 16px;
+}
+
+.price__title {
+  color: var(--gray);
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 400;
+  margin: 0;
+  line-height: 18px;
+}
+
+.cost {
+  color: var(--dark-blue);
+  text-align: right;
+  font-size: 16px;
+  font-style: normal;
+  margin: 0;
+  font-weight: 700;
+  line-height: 18px;
+}
+
+.button {
+  border-radius: 5px;
+  background: #00ACDC;
+  height: 30px;
+  display: block;
+  margin: 16px;
+  border: var(--blue);
+  color: var(--white);
+  text-align: center;
+  border-color: var(--white);
+  font-size: 14px;
+  font-style: normal;
+  font-weight: 700;
+  padding-top: 10px;
+  line-height: 16px;
+  text-transform: uppercase;
+  text-decoration: none;
+}
+
+.button:hover {
+  color: #00ACDC;
+  transition: 300ms;
```

Comment:
move transition to button (not to its hover effect) and specify what exactly you'd like to change (using transition-property)